cvat/apps/onepanelio/urls.py

```diff
@@ -16,4 +24,6 @@
     path('get_available_dump_formats', views.get_available_dump_formats),
     path('get_output_path/<int:pk>', views.generate_output_path),
     path('get_annotation_path/<int:pk>', views.generate_dataset_path),
+
+    path('api/opv1/', include(router.urls))
```

Comment:
does this match our previous naming conventions for routes?

Reply:
I was trying to mimic CVAT's engine urls, but I'll do the change pointed out in the comment below.